fdt/items.py

```diff
@@ -41,11 +42,10 @@ def new_property(name: str, raw_value: bytes) -> object:
     elif len(raw_value) and len(raw_value) % 4 == 0:
         obj = PropWords(name)
         # Extract words from raw value
-        for i in range(0, len(raw_value), 4):
-            obj.append(unpack(">I", raw_value[i:i + 4])[0])
+        obj.data = [BIGENDIAN_WORD.unpack(raw_value[i:i + 4])[0] for i in range(0, len(raw_value), 4)]
         return obj
 
-    elif len(raw_value) and len(raw_value):
```

Comment:
This is not a necessary part of the change - it just seemed very odd, so I cleaned it up.